src/net.cpp

```diff
@@ -2837,13 +2835,12 @@ void CConnman::PushMessage(CNode* pnode, CSerializedNetMsg&& msg)
         if (message_size) {
             pnode->vSendMsg.push_back(std::move(msg.data));
         }
-
-        // If write queue empty, attempt "optimistic write"
-        if (optimistic_send) {
-            bytes_sent = SocketSendData(pnode);
-        }
     }
-    if (bytes_sent) RecordBytesSent(bytes_sent);
+    // If write queue was empty, attempt "optimistic write"
+    if (optimistic_send == true) {
+        size_t bytes_sent{SocketSendData(pnode)};
+        if (bytes_sent) RecordBytesSent(bytes_sent);
+    }
```

Comment:
e85fb2d34d8515513e404163cf149be61e1ed708
```suggestion
    if (optimistic_send) {
        const size_t bytes_sent = SocketSendData(pnode);
        if (bytes_sent) RecordBytesSent(bytes_sent);
    }
```

Reply:
Thanks for catching the `const` on this one.  However, I prefer list initialization whenever possible, so I'm going to keep it unless you feel strongly against it.

Reply:
I have a mild stylistic preference to use `=` when initializing a simple type with the return value of a function, and to use braced initialization for initializing from a literal. That's perhaps inconsistent, but it seems to be the style that most people gravitate towards. No strong preference since they both result in the same thing, but if it's a vote, I'd go for `=` :)